lib/optimize/SplitChunksPlugin.js

```diff
@@ -611,6 +768,18 @@ module.exports = class SplitChunksPlugin {
 										? Object.create(null)
 										: this.options.maxSize
 								),
+								maxAsyncSize: mergeSizes(
+									cacheGroupSource.maxSize,
+									cacheGroupSource.enforce
+										? Object.create(null)
```

Comment:
```suggestion
										? undefined
```

That should be more efficient once `mergeSizes` ignored `undefined` values.